web_server/friendship/tests.py

```diff
@@ -1,3 +1,77 @@
-from django.test import TestCase
+from django.test import TestCase, Client
```

Comment:
I thought we had discussed using the unittest library for tests. Are you expecting us to use django tests for tests?

Reply:
Nevermind, I see now in @hyeonyu1 PR #109 that we are using django tests.